src/server/game/Entities/Player/Player.h

```diff
@@ -2761,6 +2761,8 @@ class TC_GAME_API Player : public Unit, public GridObject<Player>
 
         void SetTransportServerTime(int32 transportServerTime) { SetUpdateFieldValue(m_values.ModifyValue(&Player::m_activePlayerData).ModifyValue(&UF::ActivePlayerData::TransportServerTime), transportServerTime); }
 
+        void SetChromieTimeExpansion(uint8 chromieTimeExpansion) { SetUpdateFieldValue(m_values.ModifyValue(&Player::m_activePlayerData).ModifyValue(&UF::ActivePlayerData::UiChromieTimeExpansionID), chromieTimeExpansion); }
```

Comment:
This function can send ExpansionLevelMask too with a db2 function to get chromie time expansion info where the ID = chromieTimeExpansion argument of the function